### PR TITLE
Add effort appendix to DG and fix ModelClassDiagram

### LINKED ISSUE #239: Add effort section to appendix

### PR DESCRIPTION
**Pull request by josiahkhoo**
Included `Statistics` in our ModelClassDiagram.

Closes #239.

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1094,3 +1094,50 @@ There are 2 ways to delete flashcards – by index or by tags.
...
+4. To be able to show statistics, it was important for us to maintain a state on each flashcard, even between sessions. To solve this, we created a `Statistics` class and used it to keep track of the number of times a question is tested and is tested correctly.
+
+### Effort
+
+In terms of effort compared to AB3, we felt that QuickCache took at least the same amount of effort, or twice that of AB3 to develop. You can refer to the table below to see the additions and extensions we made to the commands that AB3 came with:
+
+| Commands | AB3 | QuickCache | Extensions |
+| --- | --- | --- | --- |
+| add | Y | Y | Add supports both Multiple Choice and Open Ended questions |
+| edit | Y | Y | |
```
Comment (GilbertTan19) [inline docs/DeveloperGuide.md]: I'm not so sure if edit can support both MCQ and OEQ is extension?

**Review by GilbertTan19 — APPROVED** [review]
Overall, LGTM!

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (josiahkhoo) [reply docs/DeveloperGuide.md]: Valid point! Thanks for pointing that out Gilbert, added it in bf9b870.

**Review by joshtyf — COMMENTED** [review]
Left a few questions on our tele chat. But thanks for putting in the effort appendix

**Pull request merged**